conda-recipe/meta.yaml

```diff
@@ -1,7 +1,4 @@
-{% set required_compiler_version = "2024.0" %}
-{% set excluded_compiler_version1 = "2024.0.1" %}
-{% set excluded_compiler_version2 = "2024.0.2" %}
-{% set excluded_compiler_version3 = "2024.0.3" %}
+{% set required_compiler_version = "2024.2.0" %}
```

Comment:
Should the tests in `libsyclinterface/tests/test_sycl_device_interface.cpp` be updated and FIXME be removed or the issue still present with 2024.2 compiler?

Reply:
Regrettably 2024.2 did not yet fix it:

```ipython
(dev_dpctl) sp@sp-mobl:~/repos/dpctl_$ ipython
Python 3.12.3 | packaged by conda-forge | (main, Apr 15 2024, 18:38:13) [GCC 12.3.0]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.24.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import dpctl, ctypes, os

In [2]: d = dpctl.select_cpu_device()

In [3]: d.native_vector_width_half, d.preferred_vector_width_half
Out[3]: (0, 0)

In [4]: si_lib = ctypes.cdll.LoadLibrary(os.path.join(dpctl.get_include(), "..", "libDPCTLSyclInterface.so"))

In [5]: ver_fn = si_lib.DPCTLService_GetDPCPPVersion

In [6]: ver_fn.restype, ver_fn.argtypes = ctypes.c_char_p, []

In [7]: ver_fn()
Out[7]: b'2024.2.0'
```